### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by kevmoo**
Preparing for new features

### CODE REVIEW

File: lib/src/builder_impl.dart [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+import 'generator.dart';
+import 'library.dart';
+import 'utils.dart';
+
+const defaultFileHeader = '''// GENERATED CODE - DO NOT MODIFY BY HAND
+
+''';
+
+/// A [Builder] wrapping on one or more [Generator]s.
+class BuilderImpl extends Builder {
```
Comment (natebosch) [inline lib/src/builder_impl.dart]: why does this need to be in a different file?

why does this need to be named like "Impl"?

**Comment by kevmoo** [comment]
Because that's the way *uh huh uh huh* I like it. *uh huh uh huh*

File: lib/src/builder_impl.dart (hunk 1 above)
Comment (matanlurey) [reply lib/src/builder_impl.dart]: +1. This change isn't valuable.

**Comment by kevmoo** [comment]
Ack all around – @natebosch @matanlurey – PTAL

**Pull request merged**